app/src/main/java/com/example/tmdb/PaginationScrollListener.java

```diff
@@ -0,0 +1,51 @@
+package com.example.tmdb;
+
+
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+/**
+ * Pagination
+ * Created by Suleiman19 on 10/15/16.
+ * Copyright (c) 2016. Suleiman Ali Shakir. All rights reserved.
```

Comment:
Dont blindly copy anything from internet, take reference and implement in your way